node/resolvers/queries/index.ts

```diff
@@ -41,15 +35,7 @@ const buildWhereStatement = async ({
     const orgArray = [] as string[]
 
     organization.forEach(async (org) => {
```

Comment:
Also, it looks preferable to use .map instead of a .forEach and then a push

Reply:
Done, thanks!